# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/140-math_decl.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

https://dev.openwrt.org/browser/trunk/toolchain/uClibc/patches-0.9.30.1/140-math_decl.patch
--- ./libc/sysdeps/linux/common/bits/mathcalls.h.orig
+++ ./libc/sysdeps/linux/common/bits/mathcalls.h
@@ -229,7 +229,7 @@
 
 
 /* Return nonzero if VALUE is not a number.  */
-__MATHDECL_PRIV (int,__isnan,, (_Mdouble_ __value), (__const__));
+__MATHDECL_PRIV (int,isnan,, (_Mdouble_ __value), (__const__));
 
 #if defined __USE_MISC || defined __USE_XOPEN
 /* Return nonzero if VALUE is not a number.  */
